lib/include/event-driven/vDraw.h

```diff
@@ -1228,100 +181,15 @@ class skinDraw : public vDraw {
 
 class skinsampleDraw : public vDraw {
 
-
-private:
-
-    loadMap tmap;
-    cv::Mat baseimage;
-    int radius,noise,max_value;
-
 public:
-  void initialise(){
 
-        Xlimit = 800;
-        Ylimit = 800;
-
-        baseimage = cv::Mat(cv::Size(Xlimit, Ylimit), CV_8UC3);
-        baseimage.setTo(255);
-
-        tmap.initialise("leftHand");
-
-        radius = tmap.radius;
-        noise = tmap.noise;
-        max_value=tmap.max_value;
-
-        int x;
-        int y;
-
-        
-        for (auto it = tmap.pos.begin(); it != tmap.pos.end(); it++)
-        {
-            x = tmap.xoffset + tmap.scaling* std :: get<0>(it->second);
-            y =  tmap.yoffset +tmap.scaling* std :: get<1>(it->second);
-            cv::Point centr_all(x, y);
-            cv::circle(baseimage, centr_all, radius, black,1, CV_AA);
-        }
-    };
-
-
-
-    virtual void resetImage(cv::Mat &image){
-        baseimage.copyTo(image);
-    };
     static const std::string drawtype;
     virtual void draw(cv::Mat &image, const ev::vQueue &eSet, int vTime);
     virtual std::string getDrawType();
     virtual std::string getEventType();
 
 };
 
-
-class taxelsampleDraw : public vDraw {
-
-    public:
-    void resetImage(cv::Mat &image){
-        if(image.empty()) {
-        image = cv::Mat(Ylimit, Xlimit, CV_8UC3);
-        image.setTo(0);
-        }
-    };
-    void initialise(){
-
-        Xlimit = 200;
-        Ylimit = 300;
-      
-    };
-    static const std::string drawtype;
-    virtual void draw(cv::Mat &image, const ev::vQueue &eSet, int vTime);
-    virtual std::string getDrawType();
-    virtual std::string getEventType();
-
-};
-
-class taxeleventDraw : public vDraw {
-   
-    public:
-    virtual void resetImage(cv::Mat &image){
-        if(image.empty()) {
-        image = cv::Mat(Ylimit, Xlimit, CV_8UC3);
-        image.setTo(0);
-        }
-    };
-    void initialise(){
-
-        Xlimit = 200;
-        Ylimit = 300;
-       
-    };
-    static const std::string drawtype;
-    virtual void draw(cv::Mat &image, const ev::vQueue &eSet, int vTime);
-    virtual std::string getDrawType();
-    virtual std::string getEventType();
-
-};
-
-
-
```

Comment:
Is this dead code being removed? Didn't find replacements being added in any other files